submissions/Harnytskyi/memory_pair_game/script.js

```diff
@@ -0,0 +1,83 @@
+const CARDS = ["Kyiv", "Kharkiv", "Lviv", "Odesa", "Dnipro", "Poltava", "Zhytomyr", "Khmelnytskyi", "Luhansk", "Rivne", "Cherkasy", "Mykolaiv", "Simferopol", "Zaporizhzhia"];
```

Comment:
Are you sure that naming for this array must be `CARDS` ?
Maybe `CITIES` more meaningful ?